.idea/codeStyles/Project.xml

```diff
@@ -68,6 +68,31 @@
     <MarkdownNavigatorCodeStyleSettings>
       <option name="RIGHT_MARGIN" value="72" />
     </MarkdownNavigatorCodeStyleSettings>
+    <Objective-C-extensions>
```

Comment:
Please remove changes in non-md files.

Reply:
Thanks for your review, I will remove changes of non-md files. Thanks.